### PR TITLE
Update Observability BBEs for SLP9 with Config Changes

### PR DESCRIPTION
**Pull request by madushajg**

### CODE REVIEW

File: examples/counter-metrics/counter_metrics.description [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // Ballerina supports Observability out of the box and Metrics is one of the three important aspects of 
-// Observability. To observe Ballerina code, the build time flag `--observability-included` and the runtime flag
-// `--b7a.observability.metrics.enabled=true` should be given when starting the service.
+// Observability. To observe Ballerina code, the build time flag `--observability-included` should be given along with the
+// Config.toml file when starting the service. The Config.toml file contains the required runtime configurations related to observability.
```
Comment (praneesha) [inline examples/counter-metrics/counter_metrics.description:3]:
```suggestion
// `Config.toml` file when starting the service. The `Config.toml` file contains the required runtime configurations related to observability.
```

File: examples/gauge-metrics/gauge_metrics.description [hunk 2]
```diff
@@ -1,6 +1,6 @@
 // Ballerina supports Observability out of the box and Metrics is one of the three important aspects of the
-// Observability. To observe Ballerina code, the build time flag `--observability-included` and the runtime flag
-// `--b7a.observability.metrics.enabled=true` should be given when starting the service.
+// Observability. To observe Ballerina code, the build time flag `--observability-included` should be given along with the
+// Config.toml file when starting the service. The Config.toml file contains the required runtime configurations related to observability.
```
Comment (praneesha) [inline examples/gauge-metrics/gauge_metrics.description:3]:
```suggestion
// `Config.toml` file when starting the service. The `Config.toml` file contains the required runtime configurations related to observability.
```

File: examples/tracing/tracing.description [hunk 3]
```diff
@@ -1,6 +1,6 @@
-// Ballerina supports Observability out of the box, and Tracing is one of the three important aspects of 
-// Observability. To observe Ballerina code, the build time flag `--observability-included` and the runtime flag
-// `--b7a.observability.tracing.enabled=true` should be given when starting the service.
+// Ballerina supports Observability out of the box, and Tracing is one of the three important aspects of
+// Observability. To observe Ballerina code, the build time flag `--observability-included` should be given along with the
+// Config.toml file when starting the service. The Config.toml file contains the required runtime configurations related to observability.
```
Comment (praneesha) [inline examples/tracing/tracing.description:3]:
```suggestion
// `Config.toml` file when starting the service. The `Config.toml` file contains the required runtime configurations related to observability.
```

File: examples/counter-metrics/counter_metrics.description [hunk 4]
```diff
@@ -1,6 +1,6 @@
 // Ballerina supports Observability out of the box and Metrics is one of the three important aspects of 
-// Observability. To observe Ballerina code, the build time flag `--observability-included` and the runtime flag
-// `--b7a.observability.metrics.enabled=true` should be given when starting the service.
+// Observability. To observe Ballerina code, the build time flag `--observability-included` should be given along with the
```
Comment (praneesha) [inline examples/counter-metrics/counter_metrics.description:2]:
```suggestion
// Observability. To observe Ballerina code, the `--observability-included` build time flag should be given along with the
```

File: examples/counter-metrics/counter_metrics.server.out [hunk 5]
```diff
@@ -1,9 +1,9 @@
 # To start the service, navigate to the directory that contains the
-# `.bal` file and execute the `bal run` command below with the build time flag `--observability-included` and the runtime flag `--b7a.observability.metrics.enabled=true`.
-bal run --observability-included counter_metrics.bal --b7a.observability.metrics.enabled=true
+# `.bal` file and execute the `bal run` command below with the build time flag `--observability-included` and the runtime configuration file `Config.toml`.
```
Comment (praneesha) [inline examples/counter-metrics/counter_metrics.server.out:2]:
```suggestion
# `.bal` file and execute the `bal run` command below with the `--observability-included` build time flag and the `Config.toml` runtime configuration file.
```
Please correct in all occurrences.

File: examples/counter-metrics/counter_metrics_config.out [hunk 6]
```diff
@@ -0,0 +1 @@
+# Create new .toml file `Config.toml` and add the above configurations.
```
Comment (praneesha) [inline examples/counter-metrics/counter_metrics_config.out:1]:
```suggestion
# Create a new `Config.toml` file and add the above configurations.
```

Comment (madushajg) [reply examples/counter-metrics/counter_metrics_config.out]: Fixed with 3c257fc977bdca22f195e0a54e89aff023695e39

File: examples/counter-metrics/counter_metrics.server.out (hunk 5 above)
Comment (madushajg) [reply examples/counter-metrics/counter_metrics.server.out]: Fixed with 3c257fc977bdca22f195e0a54e89aff023695e39

File: examples/counter-metrics/counter_metrics.description (hunk 4 above)
Comment (madushajg) [reply examples/counter-metrics/counter_metrics.description]: Fixed with 3c257fc977bdca22f195e0a54e89aff023695e39

File: examples/tracing/tracing.description (hunk 3 above)
Comment (madushajg) [reply examples/tracing/tracing.description]: Fixed with 3c257fc977bdca22f195e0a54e89aff023695e39

File: examples/gauge-metrics/gauge_metrics.description (hunk 2 above)
Comment (madushajg) [reply examples/gauge-metrics/gauge_metrics.description]: Fixed with 3c257fc977bdca22f195e0a54e89aff023695e39

File: examples/gauge-metrics/gauge_metrics.server.out [hunk 7]
```diff
@@ -1,9 +1,9 @@
 # To start the service, navigate to the directory that contains the
-# `.bal` file and execute the `bal run` command below with the build time flag `--observability-included` and the runtime flag `--b7a.observability.metrics.enabled=true`.
-bal run --observability-included gauge_metrics.bal --b7a.observability.metrics.enabled=true
+# `.bal` file and execute the `bal run` command below with the `--observability-included` build time flag and the `Config.toml` runtime configuration file.
+BALCONFIGFILE=Config.toml bal run --observability-included gauge_metrics.bal
 
-ballerina: enabled prometheus metrics reporter
 [ballerina/http] started HTTP/WS listener 0.0.0.0:9797
+ballerina: enabled prometheus metrics reporter
```
Comment (nadundesilva) [inline examples/gauge-metrics/gauge_metrics.server.out:6]: This should be updated to the latest changes in Prometheus

Comment (madushajg) [reply examples/gauge-metrics/gauge_metrics.server.out]: Fixed with 6469f7b3e41b70893af628ad972f36ae59abad70

**Review by praneesha — APPROVED** [review]

**Review by nadundesilva — APPROVED** [review]
LGTM

**Pull request merged**